README.md

```diff
@@ -57,95 +57,224 @@ Some of the skills/concepts this assignment aims to test (and build upon):
 * Routing (URL fragment based routing)
 
 ## API
-We can build API documentation using [Swagger](https://swagger.io/) or API Blueprint. 
+We can build API documentation using [Swagger](https://swagger.io/) or API Blueprint.
 A simple express JS API server can be built that is provided to students to allow them to develop offline. It could also be hosted somewhere to ease development when online?
 
 TODO: Someone will need to build a API server to support all of this client side functionality.
 
 The following specification is a WIP and only includes some endpoints.
 
+> POST `/login`
+
 ```json
-POST /login
 {
   "username": "string",
   "password": "string",
 }
+```
+
 returns:
+  - `200` OK for successful login, sets a cookie with a session token
+  - `403` for invalid credentials
+
+
+> POST `/signup`
 
-200 OK for successful login
-403 for invalid credentials
-POST /signup
+```json
 {
   "displayName": "string",
   "username": "string",
   "password": "string",
 }
-returns
+```
+
+> POST `/feed`
```

Comment:
This fulfills the semantics of a GET not a post.
A POST is always used for the creation of a new entry.

Reply:
Using a message body with GET is generally discouraged. Shifting to URL parameter based request.

Reply:
No body necessary. get `/feed` will be resolved by HTTP-cookie. Isn't it always the current users feed?

Reply:
We need to provide some sort of way to allow for students to request n items unless each request returns the entire feed

Reply:
`/feed?p=2`

`/feed` provides say 10 most recent?

Reply:
Then each p, just gives the next batch?

Reply:
Yep that's the current system.

Reply:
Maybe /feed returns all. WHich we'll allow for people who can't deal with small request bundles in their logic. Then p=1,2,3,4...n is each batch of 10.

Reply:
hmm yeah ok i'm ok with changing /feed to be all posts
Also the current "position token" i have in the spec fulfils what you want there, i'll just change it from a string to a integer.